system-variables.md

```diff
@@ -193,6 +243,20 @@ mysql> SELECT * FROM t1;
 - Default value: `Apache License 2.0`
 - This variable indicates the license of your TiDB server installation.
 
+### max_allowed_packet
+
+- Scope: GLOBAL | SESSION
+- Default value: `67108864`
+- Range: `[1024, 1073741824]`
+- Maximum size of a packet for the MySQL protocol
+
+### max_connections
+
+- Scope: GLOBAL
+- Default value: `151
+- Range: `[1, 100000]`
+- MySQL Compatibility variable. Not used in TiDB.
```

Comment:
It looks like this should be a noop sysvar (there is a concept of max connections in tidb; but it uses the config file to set).

So far we've not been documenting noop sysvars. I think until we've documented all the actual sysvars we should stick to this strategy, otherwise we'll get tied down in reviews for documentation that is not useful.